src/io/gff.rs

```diff
@@ -182,7 +183,7 @@ impl<W: io::Write> Writer<W> {
             record
                 .attributes
                 .iter()
-                .map(|(a, b)| format!("{}{}{}", a, self.delimiter, b))
+                .map(|item| format!("{}{}{}", item.0, self.delimiter, item.1))
```

Comment:
Oh wait, this should not be here. I think I was playing with something else when I was still using `Map<String, Vec<_>>`. I'll change this soon.